examples/src/main/python/sql/arrow.py

```diff
@@ -44,12 +46,11 @@ def dataframe_with_arrow_example(spark):
 
     # Convert the Spark DataFrame back to a Pandas DataFrame using Arrow
     result_pdf = df.select("*").toPandas()
-    # $example off:dataframe_with_arrow$
+
     print("Pandas DataFrame result statistics:\n%s\n" % str(result_pdf.describe()))
 
 
 def ser_to_frame_pandas_udf_example(spark):
-    # $example on:ser_to_frame_pandas_udf$
     import pandas as pd
 
     from pyspark.sql.functions import pandas_udf
```

Comment:
`pyspark.sql.pandas.functions`?

Reply:
Actually `pyspark.sql.pandas` is kind of an internal module. It is exposed through `pyspark.sql.functions`.